roles/oem/tasks/vm_only_pre.yml

```diff
@@ -1,6 +1,19 @@
 ---
 
 # Tasks for the oem role to run at the beginning of playbook execution
+- name: Gather the package facts
+  ansible.builtin.package_facts:
+    manager: auto
+- name: Remove unneeded backgrounds
+  ansible.builtin.apt:
+    name: '{{ item }}'
+    state: absent
+    purge: yes
+  loop: "{{ ansible_facts.packages | select('search', 'mint-background') | list }}"
```

Comment:
Can we pass this directly to `name` instead of needing to `loop`? iirc `ansible.builtin.apt` is supposed to be smart so maybe it doesn't matter either way.